lib/makara/middleware.rb

```diff
@@ -37,7 +37,7 @@ def call(env)
     # ignore asset paths
     # consider allowing a filter proc to be provided in an initializer
     def ignore_request?(env)
-      if defined?(Rails)
+      if defined?(Rails) && Rails.try(:application).try(:config).try(:assets).try(:prefix)
```

Comment:
This gem works in Sinatra an other Rack apps but we added Rails checks for that common case.